lib/hammer_cli_foreman/commands.rb

```diff
@@ -202,6 +202,9 @@ def send_request
         },
         e.resource
       )
+    rescue Exception => e
+      HammerCLIForeman.foreman_api_connection.api.clear_credentials if e.to_s.include?"401"
```

Comment:
While I think it over again it would be better to catch RestClient::Unauthorized and preferably do it in Hammer core (HammerCLI::Apipie::Command). What do you think?